8.0/debian/buster/developer/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM php:8.0-buster
+MAINTAINER "Bence Gazder"
+LABEL maintainer="bence.gazder@apicore.de"
+
+ENV COMPOSER_VERSION 2.0.11
+
+# install dependencies
+RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini" && \ 
+    apt-get update && apt-get install -y \
+    supervisor \
+    cron \
+    ca-certificates \
+    libfreetype6 \
+    libfreetype6-dev \
+    libssl1.0 \
+    libpng16-16 \
+    libjpeg62-turbo \
+    libjpeg62-turbo-dev \
+    mcrypt \
+    libcurl4-openssl-dev \
+    libpng-dev \
+    libicu-dev \
+    libonig-dev \
+    zip \
+    libzip-dev \
+    $PHPIZE_DEPS \
+    nano \
+    git \
+    openssh-client \
+    sudo \
+# configure extensions
+    && docker-php-ext-configure intl --enable-intl \
+    && docker-php-ext-configure gd --with-freetype --with-jpeg \
+# install extensions
+    && docker-php-ext-install \
+    mbstring \
+    intl \
+    curl \
+    pdo_mysql \
+    exif \
+    zip \
+    gd \
+    bcmath \
+    opcache \
+    pcntl \
+    sockets \
+# pecl extensions
```

Comment:
What do you think about including the redis extension by default? It is only a few megabytes and allows for redis support without additional layers in the final image.

The same applies to the other images as well.

Reply:
Added redis to 7.4 and 8.0 images